app/assets/sass/design-tabless.scss

```diff
@@ -0,0 +1,1035 @@
+}
+.footer-body p {
+  font-size: 19px;
+  margin-bottom: -30px;
+}
+
+
+.content-body p {
+  font-size: 19px;
+}
+
+.content-body a {
+  text-decoration: none;
+}
+
+.content-body li {
+  font-size: 22px;
+  line-height: 200%;
+}
+
+
+
+// search sort //
+
+.sort {
+  display: inline-block;
+  padding: 15px 15px 5px 0px;
+}
+
+.sort-selected {
+  background: $govuk-blue;
+  color: white;
+  padding: 10px;
+}
+
+.sort a, a.visited {
+  text-decoration: underline !important;
+}
+
+
+// option select //
+
+
+.govuk-option-select {
+ background-color: #dee0e2;
+ padding: 0 5px 5px 5px;
+ margin: 30px 0 30px 0;
+}
+
+  .govuk-option-select .filter-field-text {
+    @include core-16;
+    @include box-sizing(border-box);
+    width: 100%;
+    padding: 5px;
+    border: 1px solid $border-colour;
+  }
+
+  .govuk-option-select .option-select-label {
+      font-family: "nta", Arial, sans-serif;
+      font-size: 19px;
+      line-height: 1.31579;
+      font-weight: 400;
+      text-transform: none;
+      margin-right: 40px;
+      font-weight: bold;
+      padding-top: 5px;
+  }
+
+  .govuk-option-select .options-container {
+      max-height: 200px;
+  }
+  .govuk-option-select .options-container {
+      position: relative;
+      background-color: #fff;
+      overflow-y: scroll;
+      overflow-x: hidden;
+}
+
+.govuk-option-select .options-container input {
+    float: left;
+    margin: 2px 0 0 -27px;
+    width: 20px;
+    height: 20px;
+  }
+
+  .options-container label {
+    display: -moz-inline-stack;
+    display: inline-block;
+    font-family: "nta", Arial, sans-serif;
+    font-size: 16px;
+    line-height: 1.25;
+    font-weight: 400;
+    text-transform: none;
+    padding: 7px 0 7px 35px;
+    border-bottom: 1px solid #bfc1c3;
+    width: 100%;
+    cursor: pointer;
+  }
+
+//
+
+
+
+
+
+
+.show-all {
+  display: inline-block;
+  padding: 0px 0px 0px 0px;
+}
+
+.show-all a {
+  font-weight: bold;
+  text-decoration: none;
+}
+
+.views-block {
+  display: inline-block;
+  width: 100%;
+  margin: 10px 0px 50px 0;
+  padding-bottom: 0px;
+  border-bottom: 3px solid $grey-3;
+}
+
+.views-block li {
+ margin-right: 45px;
+ padding-bottom: 10px;
+ font-size: 24px;
+ text-decoration: none;
+ display: inline-block;
+
+}
+
+.views-block a {
+  text-decoration: none;
+}
+
+
+
+.sib {
+  padding: 5px;
+}
+
+.sibling-block-top-sec a {
+text-decoration: none;
+}
+
+.sibling-block-top-sec-2 a {
+text-decoration: none;
+}
+
+.sibling-block-top-sec-2 li {
+  padding: 0px 30px 20px 0px;
+}
+
+
+
+.sibling-block-top-sec li {
+  padding: 0px 30px 12px 0px;
+}
+
+
+.sibling-block-rhs li {
+  padding: 0px 30px 8px 15px;
+}
+
+.sibling-block-rhs a {
+  text-decoration: none;
+  display: inline-block;
+  padding: 10px 30px 15px 15px;
+  line-height: 1.14286;
+  position: relative;
+}
+
+.sibling-block-rhs ul a:after {
+  position: absolute;
+  top: 50%;
+  margin-top: -8px;
+  right: 10px;
+  float: right;
+  /*
+    content: "\203A";
```

Comment:
Probably, but i'd suggest largely ignoring the css for now. It's very much in flux and we've deliberately not worried too much about architecting it cleanly.